include/eve/module/core/regular/is_less.hpp

```diff
@@ -22,16 +23,23 @@
 namespace eve
 {
   template<typename Options>
-  struct is_less_t : strict_elementwise_callable<is_less_t, Options, definitely_option>
+  struct is_less_t : elementwise_callable<is_less_t, Options, definitely_option>
   {
-    template<value T,  value U>
-    requires(eve::same_lanes_or_scalar<T, U>)
-    constexpr EVE_FORCEINLINE common_logical_t<T,U> operator()(T a, U b) const
+    template<floating_value T, floating_value U>
+    constexpr EVE_FORCEINLINE common_logical_t<T, U> operator()(T a, U b) const
+      requires (compatible_arithmetic_values<T, U> && Options::contains(definitely))
```

Comment:
I'd just static assert in the operator().
We had some examples of that somewhere.

Also - what about the tuple types? @jfalcou

Reply:
The question is : 
 - do we want SFINAEing out people use is_less[tolerance](int,int)
 - or do we want an hard error ?

Reply:
Depends on what you want for tuple

Reply:
Well we need a global protocol. Is using options when they aren't supposed too sfinae or hard error.

Reply:
I think the answer will come from what you want for tuple. Can you look at that first please